src/db/awsES/locations-dao.js

```diff
@@ -1,29 +1,21 @@
-import _ from 'lodash';
-import aws from 'aws-sdk';
-import config from 'config';
-import connectionClass from 'http-aws-es';
-import elasticsearch from 'elasticsearch';
+import { Client } from 'elasticsearch';
 import esb from 'elastic-builder';
+import _ from 'lodash';
 
+import { clientOptions } from 'db/awsEs/connection';
 import { parseQuery } from 'utils/parse-query';
 
-const {
-  domain,
-  accessKeyId,
-  secretAccessKey,
-} = config.get('dataSources.awsES');
-
 /**
  * Parses query parameters and generates an elasticsearch query body
  *
- * @param {object} queryParams Query paramaters from GET /locations request
+ * @param {object} queryParams Query parameters from GET /locations request
  * @returns {object} Elasticsearch query body
  */
 const buildQueryBody = (queryParams) => {
   const parsedParams = parseQuery(queryParams);
-  let q = esb.boolQuery();
+  let qu = esb.boolQuery();
```

Comment:
Hmm, I'm fine with this, but I think usually ppl are using `q` as variable name for query. Is there any specific reason that you wanna change to `qu` here?

Reply:
I changed to `qu` so that the dot functions on a newline will align: https://github.com/osu-mist/locations-api/pull/71#discussion_r462523616

I'm fine if you think I should just keep it as q.